GOVERNANCE.md

```diff
@@ -0,0 +1,149 @@
+# Rez Project Roles and Responsibilities
+
+Rez is a project of the Academy Software Foundation and relies on
+the ASWF governance policies, supported by the Linux Foundation.
+
+There are three primary project roles: 
+
+* [Contributors](#Contributors) submit code to the project
+* [Committers](#Committers) approve code to be included into the project
+* [Technical Steering Committee (TSC)](#Technical-Steering-Committee) 
+  members provide overall high-level project guidance.
+
+
+## Contributors
+
+The Rez project grows and thrives from assistance from
+Contributors.  Contributors include anyone in the community that
+submits code, documentation, or other technical artifacts to the
+project. However, such contributions must be approved by a project
+Committer before they become a part of the project.  
+
+Anyone can be a Contributor. You need no formal approval from the
+project, beyond the legal forms.
+
+### How to Become a Contributor
+
+* Review the coding standards to ensure your contribution is in line
+  with the project's coding and styling guidelines.
+
+* Sign the Individual CLA, or have your organization sign the Corporate CLA.
+
+* Submit your code as a PR with the appropriate DCO sign-off.
+
+## Committers
+
+Project Committers have merge access on the Rez GitHub repository
+and are responsible for approving submissions by Contributors.
+
+### Committer Responsibilities
+
+Typical activities of a Committer include:
+
+* Helping users and novice contributors.
+
+* Ensuring a response to questions posted to the ASWF rez slack channel.
+
+* Contributing code and documentation changes that improve the project.
+
+* Reviewing and commenting on issues and pull requests.
+
+* Ensuring that changes and new code meet acceptable standards and are
+  in the long-term interest of the project.
+
+* Participation in working groups.
+
+* Merging pull requests.
+
+### How to Become a Committer
+
+Any existing Committer can nominate an individual making significant
+and valuable contributions to the Rez project to become a new
+Committer.  New committers are approved by vote of the TSC.
+
+If you are interested in becoming a Committer, contact the TSC at ???
+
+TODO: best way to contact the TSC? Publicly? Privately?
+
+## Technical Steering Committee
+
+The Technical Steering Committee (TSC) oversees the overall technical
+direction of Rez, as defined in the project
+[charter](ASWF/charter/Rez-Technical-Charter.md).  This
+charter defines the TSC member terms and succession policies.
+
+TODO: Is there a rez charter? I don't know whether it belongs here or in a
```

Comment:
John started us off with this, which I believe we can modify in the future if necessary.
https://github.com/AcademySoftwareFoundation/foundation/blob/main/project_charters/rez-charter.pdf

Reply:
Indeed the link @maxnbk provided is our charter. But I don't think the charter is that easily modifiable, but I might be mistaken.

Reply:
added link, erased todo